src/theme/components/globalMixins.ts

```diff
@@ -12,5 +12,13 @@ const wrapperStyles = {
   paddingLeft: "0",
   width: "100%",
 };
+const screenreaderOnly = {
```

Comment:
Seeing this made me think of something that we should be aware of: `VisuallyHidden` from Chakra.

https://chakra-ui.com/docs/disclosure/visually-hidden

I don't think it is applicable here, however.

Reply:
Yes, I saw that too but you're right it's not exactly applicable here since we want to toggle whether its visually hidden or not.